infrastructure/main.tf

```diff
@@ -15,6 +15,14 @@ data "vault_generic_secret" "tests_s2s_secret" {
   path = "secret/${var.vault_section}/ccidam/service-auth-provider/api/microservice-keys/send-letter-tests"
 }
 
+data "vault_generic_secret" "encryption_public_key" {
+  path = "secret/${var.vault_section}/cc/send-letter/encryption-key"
```

Comment:
does such a path actually exist?

Reply:
Yes as per Leszek's update in JIRA this path exists excluding prod.Jira ticket is already in place for prod.